api-spec/wallet-api.yaml

```
@@ -614,6 +614,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
+  /wallets/{walletId}/usage:
```

[conventions]
what about
```suggestion
  /wallets/{walletId}/usages:
```
?